appserver/admingui/commandrecorder/src/main/java/org/glassfish/commandrecorder/admingui/CommandRecorder.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package org.glassfish.commandrecorder.admingui;
+
+import static org.glassfish.extras.commandlogger.AdminCommandLogger.LogMode.NO_COMMAND;
+import static org.glassfish.extras.commandlogger.AdminCommandLogger.LogMode.WRITE_COMMANDS;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import java.io.Serializable;
+import javax.security.auth.Subject;
+import org.glassfish.api.ActionReport;
+import org.glassfish.api.admin.CommandRunner;
+import org.glassfish.api.admin.ParameterMap;
+import org.glassfish.extras.commandlogger.AdminCommandLogger;
+
+@Named
+@RequestScoped
+public class CommandRecorder implements Serializable {
```

Comment:
I'm getting:
- Class `org.glassfish.commandrecorder.admingui.CommandRecorder` defines non-transient non-serializable instance field
   - commandRunnerprovider
   - actionReportProvider
   - subjectProvider

Reply:
Thanks for spotting this. This is a leftover from my work in progress, I forgot to delete the `implements Serializable` clause. I'll remove it.